### PR TITLE
Sample Data Util + javadocs

### PR DESCRIPTION
**Pull request by Chesterwongz**

### CODE REVIEW

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -110,13 +119,13 @@
...
                     new School("NYJC"), EMPTY_ACAD_STREAM, EMPTY_ACAD_LEVEL,
-                    EMPTY_REMARK, getTagSet("zoom"),
+                    EMPTY_REMARK, getTagSet("J2"),
                     getLessonSet(getSampleLessons()[3])),
             new Person(new Name("David Li"), new Phone("91031282"), new Email("lidavid@example.com"),
                     new Phone("91009182"), EMPTY_EMAIL,
                     new Address("Blk 436 Serangoon Gardens Street 26, #16-43"),
                     EMPTY_SCHOOL, EMPTY_ACAD_STREAM, EMPTY_ACAD_LEVEL,
-                    new Remark("Weak at trigo"), getTagSet("zoom"),
+                    new Remark("Weak at trigo"), getTagSet("SEC2", "OLVL"),
```
Comment (lingshanng) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:128]: not sure if it's a good idea to put the acad level as a tag. And anyway why sec 2 taking o level hahaha

Comment (Chesterwongz) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java:128]: @lingshanng i think its fine thoooo

Comment (eeliana) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java]: I agree with Lingshan. Use the acad level field for acad level at least on our side. If the user wants to do this, then that's up to them.

Comment (Xiaoyunnn) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java]: eh why need change sample data ;-; if this changes, I need redo all the annotations for ug screenshots again

Comment (eeliana) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java]: Maybe make disclaimer that tags may differ if don't want to change? I think some differences in sample data probably not considered a bug

**Review by eeliana — APPROVED** [review]
LGTM

**Review by lingshanng — APPROVED** [review]

**Pull request merged**